synfig-core/src/synfig/threadpool.h

```diff
@@ -98,6 +99,8 @@ class ThreadPool {
 	void enqueue(const Slot &slot);
 	void wait(std::condition_variable &cond, std::unique_lock<std::mutex>& lock);
 
+	void set_threads(int no_core);
```

Comment:
Please rename the method to `set_num_threads` ('num' from 'number'): let the method name descriptive in order to help code reading.

And rename `no_core` to `num_cores`or something like that: 'no' means 'without' here